README.md

```diff
@@ -140,43 +140,44 @@ Credits
 
 Owner
 
-- Guangcong Luo [Zarel] - Development, Design, Sysadmin
+- [Guangcong Luo](https://github.com/Zarel) - [Zarel] - Development, Design, Sysadmin
 
 Staff
 
-- Chris Monsanto [chaos] - Sysadmin
-- Leonardo Julca [Slayer95] - Development
-- Mathieu Dias-Martins [Marty-D] - Research (game mechanics), Development
-- [The Immortal] - Development
+- Chris Monsanto - [chaos] - Sysadmin
+- [Leonardo Julca](https://github.com/Slayer95) - [Slayer95] - Development
+- [Mathieu Dias-Martins](https://github.com/Marty-D) - [Marty-D] - Research (game mechanics), Development
+- [The Immortal](https://github.com/TheImmortal) - Development
 
 Retired Staff
 
-- Bill Meltsner [bmelts] - Development, Sysadmin
-- Cathy J. Fitzpatrick [cathyjf] - Development, Sysadmin
-- Hugh Gordon [V4] - Research (game mechanics), Development
```

Comment:
[Huge Gordon] 
(V4Victini)? He does have a github

Reply:
Alright I'll look, didn't see them when I looked

Reply:
Don't see it

Reply:
He's https://github.com/DanUgore